pkg/assessor/manifest/manifest_test.go

```diff
@@ -401,6 +401,26 @@ func TestAddStatement(t *testing.T) {
 	}
 }
 
+func TestSensitiveVars(t *testing.T) {
+	var tests = map[string]struct {
+		cmd      string
+		expected bool
+	}{
+		"basic":              {cmd: "/bin/sh -c #(nop) ENV PASS=ADMIN", expected: true},
```

Comment:
I think it's a good idea to add a test case for mixed caps, for example `ENV PasS=ADMIN`